README.md

```diff
@@ -1,77 +1,22 @@
-# 1Password Connect
+<img src="https://github.com/1Password/connect/assets/46452606/0f7cf2a8-a290-41fc-b78d-3dfb1017f9be" alt="Header" role="img" >
```

Comment:
Someone using screen reading technology probably wouldn't find it helpful if the image was read out as "Header" to them, so we can do `alt=""` instead (i.e. set `alt` to an empty string).

Per [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attributes) - "Setting this attribute to an empty string (alt="") indicates that this image is not a key part of the content (it's decoration or a tracking pixel), and that non-visual browsers may omit it from rendering). Visual browsers will also hide the broken image icon if the alt is empty and the image failed to display."

Reply:
Good point, the term header isn't that useful in this case